taskfiles/backend.yaml

```diff
@@ -96,11 +111,21 @@ tasks:
     deps:
       - install-gomock
       - install-ifacemaker
+      - install-buf
     cmds:
       - |
         export PATH=$PATH:{{ .BUILD_ROOT }}/bin/go
         GOBIN={{ .BUILD_ROOT }}/bin/go go generate ./...
       - task: 'fmt'
+      - |
+        export PATH=$PATH:{{ .BUILD_ROOT }}/bin/go
+        {{ .BUILD_ROOT }}/bin/buf format -w pkg/kafka/testdata/proto
+      - |
+        export PATH=$PATH:{{ .BUILD_ROOT }}/bin/go
+        {{ .BUILD_ROOT }}/bin/buf lint pkg/kafka/testdata/proto
+      - |
+        export PATH=$PATH:{{ .BUILD_ROOT }}/bin/go
+        {{ .BUILD_ROOT }}/bin/buf generate pkg/kafka/testdata/proto
```

Comment:
Should we run the fmt task at the end? otherwise I imagine that the generated code will trigger linting errors